app/views/kudos/_form.html.erb

```diff
@@ -38,6 +38,11 @@
     <%= collection_select( :kudo, :receiver_id, Employee.all, :id, :email, { prompt: "Please select the email of person you want to grant a Kudo" }, { class: "form-select" }) %>
   </div>
 
+  <hr class="my-4">
+  <div class="field mb-3">
+    <%= collection_select( :kudo, :company_value_id, CompanyValue.all, :id, :title, { prompt: "Please select CV" }, { class: "form-select" }) %>
```

Comment:
I would use full name, since CV might be confusing ;)

```suggestion
    <%= collection_select( :kudo, :company_value_id, CompanyValue.all, :id, :title, { prompt: "Please select Company Value" }, { class: "form-select" }) %>
```

Reply:
I missed it. Of course you're right. Corrected!